crucible-mir/src/Mir/Generator.hs

```diff
@@ -167,7 +167,7 @@ data PtrMetadata s =
 
 instance Show (PtrMetadata s) where
     show NoMeta = "NoMeta"
-    show (SliceMeta _) = "SliceMeta"
+    show (SliceMeta len) = "SliceMeta " ++ show len
```

Comment:
Hmm indeed. Someone must have done it that way on purpose then. Wonder why... looks like the contents have changed since so it's probably no longer relevant.

Updated in d4bd0cd0ff03b11e463ef51b795d004c8eaa802e